src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/TypedElementSearchIndexerTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Hl7.Fhir.ElementModel;
+using Hl7.Fhir.Model;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Core.Extensions;
+using Microsoft.Health.Fhir.Core.Features.Definition;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
+using Microsoft.Health.Fhir.Core.Features.Search;
+using Microsoft.Health.Fhir.Core.Features.Search.Converters;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.Tests.Common;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Core.UnitTests.Features.Search
+{
+    public class TypedElementSearchIndexerTests
```

Comment:
There was a lot of work put into getting the search indexer out of "Shared" and into FHIR agnostic core, a little sad to see it back :(

Reply:
:( I can see if I can move this test to Microsoft.Health.Fhir.Core.UnitTests from Microsoft.Health.Fhir.Shared.Core.UnitTests